src/main/app/documents/documentsClient.ts

```diff
@@ -28,4 +28,21 @@ export class DocumentsClient {
     })
   }
 
+  getClaimIssueCopy (claimExternalId: string): http.IncomingMessage {
+    if (StringUtils.isBlank(claimExternalId)) {
+      throw new Error('Claim external ID cannot be blank')
+    }
+    return requestNonPromise.get({
+      uri: `${this.documentsUrl}/claimIssueReceipt/${claimExternalId}`
+    })
+  }
+  getDefendantResponseReceipt (claimExternalId: string): http.IncomingMessage {
```

Comment:
`getDefendantVersionOfResponseReceipt`
there's also a claimant version which we should be getting at some point

Reply:
I like the name change but it would make me think to change gerResponseCopy, for consistency.
getDefendantResponseReceipt -> getDefendantVersionOfResponseReceipt
getResponseCopy -> getCourtVersionOfResponseReceipt

Reply:
it isn't the court version it would be the claimant version

the claimant and staff versions are the same but staff don't need to get it from the frontend

Reply:
My mistake, from the name i thought it was the sealed claim version of the response. 
So should i just change to getDefendantVersionOfResponseReceipt or 

getDefendantResponseReceipt -> getDefendantVersionOfResponseReceipt
getResponseCopy -> getClaimantVersionOfResponseReceipt

Reply:
This file has been reworked. It now only has 1 method which takes in a document template string.